aptos-move/move-examples/hello_prover/Move.toml

```diff
@@ -3,7 +3,7 @@ name = "hello_prover"
 version = "0.0.0"
 
 [dependencies]
-AptosFramework = { git = "https://github.com/aptos-labs/aptos-core.git", subdir = "aptos-move/framework/aptos-framework/", rev = "main" }
+AptosFramework = { git = "https://github.com/aptos-labs/aptos-core.git", subdir = "aptos-move/framework/aptos-framework/", rev = "framework-mainnet" }
```

Comment:
framework-testnet/framework-mainnet are not up to date. @perryjrandall can you help update?